### PR TITLE
feature/FOROME-1401: replace old input components to  a new one

### PR DESCRIPTION
**Pull request by QSdmitrioul**
[Task](https://quantori.atlassian.net/browse/FOROME-1401)

### CODE REVIEW

**Comment by Lapk1n** [comment]
1. Hover on input mustn't increase its width and height, only change bg
2. Placeholder has a white color in the market. But I find grey is more appropriate, so the loop probably should be grey too.
3. Typing must change the loop color
Screenshot 2022-08-25 at 11 09 38

1. There are some places in the market where input labels have a 12px font
Screenshot 2022-08-25 at 11 19 16

1. Someone cut it from the left (and right too)
Screenshot 2022-08-25 at 11 20 19

**Comment by QSdmitrioul** [comment]
Fixed all comments

**Comment by Lapk1n** [comment]
1. These inputs don't  have a hover effect:
Screenshot 2022-08-26 at 10 39 28
Screenshot 2022-08-26 at 10 42 10
Screenshot 2022-08-26 at 10 43 41

**Comment by QSdmitrioul** [comment]
Yep, they don't have, because we have some issues with design(backround color wrong). I already discuss this with Anastasia, she said, that after current tasks she will switch and fixed this design, till that moment she gave permission to leave this input without hoovering style. 
P.S. They have hoover effect, but color of input in static mode is similar to background color, so in this cases all three inputs have styles as they always in hoover mode.

**Review by Lapk1n — APPROVED** [review]

File: src/ui/input/input.stories.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const Template: ComponentStory<typeof Input> = args => {
         <Input
           {...args}
           value={state}
-          onChange={e => setState(e.target.value)}
+          onChange={(e: any) => setState(e.target.value)}
```
Comment (EvgeniySobolev) [inline src/ui/input/input.stories.tsx]: We can import `ChangeEvent` from react.
So type will be `(e: ChangeEvent<HTMLInputElement>)`

**Review by EvgeniySobolev — APPROVED** [review]

File: src/ui/input/input.stories.tsx (hunk 1 above)
Comment (QSdmitrioul) [reply src/ui/input/input.stories.tsx]: fixed

**Pull request merged**